server/chat-plugins/wifi.tsx

```diff
@@ -1374,317 +1442,454 @@ function makePageHeader(user: User, pageid?: string) {
 		submitted: `View Submitted`,
 		'submitted-add': `Submit`,
 	};
-	const icons: Record<keyof typeof titles, string> = {
-		create: `<i class="fa fa-sticky-note"></i>`,
-		stored: `<i class="fa fa-paste"></i>`,
-		'stored-add': `<i class="fa fa-paste"></i>`,
-		submitted: `<i class="fa fa-inbox"></i>`,
-		'submitted-add': `<i class="fa fa-inbox"></i>`,
+	const icons: Record<keyof typeof titles, Chat.VNode> = {
+		create: <i className="fa fa-sticky-note"></i>,
+		stored: <i className="fa fa-paste"></i>,
+		'stored-add': <i className="fa fa-paste"></i>,
+		submitted: <i className="fa fa-inbox"></i>,
+		'submitted-add': <i className="fa fa-inbox"></i>,
```

Comment:
We use `class`, not `className`.

```suggestion
		create: <i class="fa fa-sticky-note"></i>,
		stored: <i class="fa fa-paste"></i>,
		'stored-add': <i class="fa fa-paste"></i>,
		submitted: <i class="fa fa-inbox"></i>,
		'submitted-add': <i class="fa fa-inbox"></i>,
```

Reply:
Like I’ve asked/implied before, please make some documentation for this. I asked about exactly this in the discord and you didn’t answer me, and now you’re telling me this in a tone like I’m supposed to know better.

Reply:
Sorry, it was meant to be neutral, but I did answer when you asked on Discord:

![image](https://user-images.githubusercontent.com/551184/147401136-8178e80f-6d21-4532-a54c-cea3ce9e5e86.png)

We use `class` because we use Preact and Preact uses `class`.

Reply:
The last line in that log was semi-implicit that you were unsure what the standard, so it confused me with what you meant.

Reply:
Yeah, I'm unsure about how Preact handles `for`. But I'm sure Preact prefers `class` over `className`.